dotcom-rendering/src/web/components/ReaderRevenueLinks.importable.tsx

```diff
@@ -345,8 +354,9 @@ const ReaderRevenueLinksNative: React.FC<{
 				<div css={subMessageStyles}>
 					<div>Available for everyone, funded by readers</div>
 				</div>
-				<PrimaryButton />
-				<SecondaryButton />
+				{!inHeader && <SupportButton />}
+				{inHeader && <PrimaryButton />}
+				{inHeader && <SecondaryButton />}
```

Comment:
I think a comment and a ternary would a little clearer about the intention, here.

```suggestion
				{/* When in the header, we can assume a remote header is being pulled in */}
				{inHeader ? (<PrimaryButton /><SecondaryButton />) : (<SupportButton />)}
```

Reply:
Agreed, I'll adjust this now!

Reply:
The linter or formatter may complain 🙇